### ISSUE TITLE
Ignore whole models / tables

### ISSUE THREAD

**Issue by apss-pohl** [unlight/prisma-nestjs-graphql#150]
Hi,

first things first: Great work!
I have a question. This there a way to completely ignore a model? 
This is useful e.g. for the prisma table that holdes the migrations. It will never be used within the application and is for internal purpose only. I could not find anything in the documentation so far..

Thank you

**Comment by unlight** [comment]
If you are not referencing this type in application it should not appear in graphql schema.

Also try `@ObjectType({ isAbstract: true })`
https://github.com/unlight/prisma-nestjs-graphql#objecttype
```
// schema.prisma
/// @ObjectType({ isAbstract: true })
model User {
    id Int @id
}
```

**Comment by apss-pohl** [comment]
Hiding it from gql is working, thanks you for the hint, and a step into the right direction.
But i would prefer not to have the files at all that are generated. 
Something into the direction of a decorator "///@Ignore" that would skip the whole table if this is possible.

**Comment by yiyi-wang-rv** [comment]
Yes it would be great if we can have `/// @ignore` to ignore the tables.
